official/resnet/keras/keras_common_test.py

```diff
@@ -66,7 +66,7 @@ def test_build_stats_sparse(self):
     self.assertEqual(1.9844, stats['eval_loss'])
 
   def test_time_history(self):
```

Comment:
not a big deal but this test should probably be in keras_utils_test although i haven't checked if that exists or not

Reply:
Yes this one could go there for sure.  Long story to even get to this point.